cli/core/test/e2e/framework/config_operations.go

```diff
@@ -0,0 +1,144 @@
+	return &configOps{
+		CmdOps: NewCmdOps(),
+	}
+}
+
+// ConfigSetFeature sets the tanzu config feature flag
+func (co *configOps) ConfigSetFeatureFlag(path, value string) (err error) {
+	confSetCmd := ConfigSet + path + " " + value
+	_, _, err = co.Exec(confSetCmd)
+	return err
+}
+
+// ConfigSetFeature sets the tanzu config feature flag
+func (co *configOps) ConfigGetFeatureFlag(path string) (string, error) {
+	out, _, err := co.Exec(ConfigGet)
+	if err != nil {
+		return "", err
+	}
+
+	var node yaml.Node
+	err = yaml.Unmarshal(out.Bytes(), &node)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to construct yaml node from config get output")
+	}
+	cnf, err := convertNodeToClientConfig(&node)
+	if err != nil {
+		return "", err
+	}
+	featureName := strings.Split(path, ".")[len(strings.Split(path, "."))-1]
+	if cnf != nil && cnf.ClientOptions.Features["global"] != nil {
+		return cnf.ClientOptions.Features["global"][featureName], nil
+	}
+	return "", err
+}
+
+// ConfigUnsetFeature un-sets the tanzu config feature flag
+func (co *configOps) ConfigUnsetFeature(path string) (err error) {
+	unsetFeatureCmd := ConfigUnset + path
+	_, _, err = co.Exec(unsetFeatureCmd)
+	return
+}
+
+// ConfigInit performs "tanzu config init"
+func (co *configOps) ConfigInit() (err error) {
+	_, _, err = co.Exec(ConfigInit)
+	return
+}
+
+// ConfigServerList returns the server list
+// TODO: should return the servers info in proper format
+func (co *configOps) ConfigServerList() (err error) {
+	_, _, err = co.Exec(ConfigServerList)
+	return
+}
+
+// ConfigServerDelete deletes a server from tanzu config
+func (co *configOps) ConfigServerDelete(serverName string) error {
+	_, _, err := co.Exec(ConfigServerDelete + serverName)
+	return err
+}
+
+// DeleteCLIConfigurationFiles deletes cli configuration files
+func (co *configOps) DeleteCLIConfigurationFiles() error {
+	homeDir, _ := os.UserHomeDir()
+	configFile := filepath.Join(homeDir, ConfigFileDir, ConfigFileName)
+	_, err := os.Stat(configFile)
+	if err == nil {
+		if ferr := os.Remove(configFile); ferr != nil {
+			return ferr
+		}
+	}
+	configNGFile := filepath.Join(homeDir, ConfigFileDir, ConfigNGFileName)
+	if _, err := os.Stat(configNGFile); err == nil {
+		if ferr := os.Remove(configNGFile); ferr != nil {
+			return ferr
+		}
+	}
+	return nil
+}
+
+// IsCLIConfigurationFilesExists checks the existence of cli configuration files
+func (co *configOps) IsCLIConfigurationFilesExists() bool {
+	homeDir, _ := os.UserHomeDir()
+	configFilePath := filepath.Join(homeDir, ConfigFileDir, ConfigFileName)
+	configNGFilePath := filepath.Join(homeDir, ConfigFileDir, ConfigNGFileName)
+	_, err1 := os.Stat(configFilePath)
+	_, err2 := os.Stat(configNGFilePath)
+	if err1 == nil && err2 == nil {
+		return true
+	}
+	return false
+}
+
+// convertNodeToClientConfig converts yaml node to client config type
+func convertNodeToClientConfig(node *yaml.Node) (obj *configapi.ClientConfig, err error) {
```

Comment:
based on the above comment, I think we can don't need this function and it can be removed.

Reply:
removed